### DISCUSSION TITLE
Openssl error with local domain

### DISCUSSION THREAD

**Discussion by manutepowa** [jellydn/hurl.nvim#192] [Q&A] [answered]
### Details
![image](https://github.com/user-attachments/assets/9070ec16-135e-4fb3-9c4b-4141e7068081)

How solve error with local domain?

Have a local domain with ssl by mkcert.

**Comment by sweep-ai-deprecated** [comment]
<div align='center'><a href="https://github.com/sweepai/sweep"><img class="swing" src="https://raw.githubusercontent.com/sweepai/sweep/main/.assets/sweeping.gif" width="100" style="width:50px; margin-bottom:10px" alt="Sweeping"></a></div><br/><div align='center'>

<img src='https://progress-bar.dev/0/?&title=Progress&width=600' alt='0%' /></div>

<div align='center'>💎 <b>Sweep Pro</b>: You have unlimited Sweep issues</div>

## Actions
- [ ] ↻ Restart Sweep
## Step 1: 🔎 Searching
I'm searching for relevant snippets in your repository. If this is your first time using Sweep, I'm indexing your repository, which will take a few minutes.

---

> [!TIP]
> To recreate the pull request, edit the issue title or description.

**Comment by jellydn** [comment]
I think this is something related to the way Hurl implements SSL. Is there something similar? https://github.com/Orange-OpenSource/hurl/issues?q=sort:updated-desc+is:issue+is:open+ssl

**Comment by manutepowa** [comment, marked as answer]
hello!!!
I solve uninstall curl with brew.

I had 2 curl installed. Brew + APT
